### PR TITLE
Go modules

### PR DESCRIPTION
**Pull request by alvarocabanas**
Switch to go modules

### CODE REVIEW

File: go.mod.old [hunk 1]
```diff
@@ -0,0 +1,33 @@
+module github.com/newrelic/nri-docker
+
+go 1.16
+
+require (
+	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 // indirect
```
Comment (paologallinaharbur) [inline go.mod.old:6]: this should not be here I guess

Comment (alvarocabanas) [reply go.mod.old:6]: Yes, I will delete it. I don't know why it was there.

Comment (alvarocabanas) [reply go.mod.old]: done!

**Review by sigilioso — APPROVED** [review]
LGTM, I think we all are going to avail of this change soon 👍 . On the other hand, I think it would be good to have an extra review from somewhere with more experience in this integration.

**Comment by sigilioso** [comment]
You can update from master to include [this](https://github.com/newrelic/nri-docker/pull/90) and remove the unsuccessful checks.

**Pull request merged**